tests/app/service/test_rest.py

```diff
@@ -1710,6 +1710,39 @@ def test_get_all_notifications_for_service_in_order(notify_api, notify_db_sessio
         assert response.status_code == 200
 
 
+@pytest.mark.parametrize('data', [
+    {},
+    {'page': 1, 'template_type': ['sms'], 'status': ['created', 'sending', 'delivered', 'failed'], 'to': '07'},
```

Comment:
I think I'd suggest this test gets split in to two please.

One for the very standard case of no arguments where we make sure it returns just the results for this service in order.

One for the more particular case where we can see that it is indeed filtering by template type, status and `to`, as at the moment although it is indeed showing sms notifications in one of the 4 statuses that start with `07`, we don't know it is working correctly because there are no emails or sms in other statuses or sms being sent to different numbers that we are checking are excluded.

It doesn't need to be super thorough and check every criteria that we filter on, but just one to show that we are indeed passing through the key filters from the POST request and doing filtering based on them.

Reply:
👍 Now split in two